README.md

```diff
@@ -235,6 +235,15 @@ like this:
 You'll probably get a more reliable experience if you use
 [forever](https://npmjs.org/package/forever) to keep the daemon running.
 
+### Arguments
+
+Execute with the config file on the command line using -c
```

Comment:
Wow, thanks for all this feedback. I will have to review/change all of
that!
mike

On Sun, Oct 18, 2015 at 8:28 PM, Alex Jordan <notifications@github.com>
wrote:

> In README.md
> <https://github.com/e14n/pump.io/pull/1114#discussion_r42329384>:
>
> > @@ -235,6 +235,15 @@ like this:
> >  You'll probably get a more reliable experience if you use
> >  [forever](https://npmjs.org/package/forever) to keep the daemon running.
> >
> > +### Arguments
> > +
> > +Execute with the config file on the command line using -c
>
> The wording of this implies that you *have* to pass the -c flag, which
> isn't true.
>
> Perhaps something like:
>
> You may specify a configuration location other than the default with the
> -c flag:
>
> The default locations don't really belong in the "Arguments" section.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e14n/pump.io/pull/1114/files#r42329384>.
>



-- 
James Michael DuPont
Kansas Linux Fest http://kansaslinuxfest.us
Free/Libre Open Source and Open Knowledge Association of Kansas
http://openkansas.us
Member of Free Libre Open Source Software Kosova http://www.flossk.org
Saving Wikipedia(tm) articles from deletion http://SpeedyDeletion.wikia.com